django/db/backends/base/schema.py

```diff
@@ -230,6 +230,17 @@ def table_sql(self, model):
             params.extend(extra_params)
             # FK.
             if field.remote_field and field.db_constraint:
+                if 1 < len(field.to_fields):
+                    if (
+                        self.sql_create_fk
+                        and self.connection.features.supports_foreign_keys
+                    ):
```

Comment:
It's not the case in SQLite, not sure why

Reply:
SQLite does have less support for migrations, so we have to emulate their behaviour in various places. Maybe this is related to that?

Reply:
After trying to get this to work, I realized we can't implement this for SQLite until composite primary keys are merged.

That's because composite foreign keys in SQLite must refer to the composite primary key.